modules/seqwish/induce/main.nf

```diff
@@ -21,18 +21,18 @@ process SEQWISH_INDUCE {
     script:
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
-    def VERSION = '0.7.2' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
+    def input = paf.join(',') // this ensures that we can actually input a comma-separated list of PAF files as required by https://github.com/nf-core/pangenome. If one wants to use this, ensure that you put a ".collect()" behind your channel. See https://github.com/nf-core/pangenome/blob/34149c6cdc19bce3a7b99f97c769d8986a8d429b/main.nf#L543 for an example.
```

Comment:
Would prefer this comment formatted over multiple lines, but perhaps that fails the linter?

Reply:
The linter is still happy ;)